experimental/DoubleAndHyperComplexes/src/Morphisms/free_resolutions.jl

```diff
@@ -136,13 +136,18 @@ augmentation_map(c::SimpleFreeResolution) = c.augmentation_map
 function betti(b::SimpleFreeResolution; project::Union{FinGenAbGroupElem, Nothing} = nothing, reverse_direction::Bool = false)
   return betti_table(b; project, reverse_direction)
 end
-function betti_table(C::AbsHyperComplex; project::Union{FinGenAbGroupElem, Nothing} = nothing, reverse_direction::Bool=false)
+function betti_table(C::AbsHyperComplex; 
+    lower_bound::Union{Int, Nothing}=(has_lower_bound(C) ? Oscar.lower_bound(C) : nothing),
+    upper_bound::Union{Int, Nothing}=(has_upper_bound(C) ? Oscar.upper_bound(C) : nothing),
```

Comment:
```suggestion
    lower_bound::Union{Int, Nothing}=(has_lower_bound(C) ? lower_bound(C) : nothing),
    upper_bound::Union{Int, Nothing}=(has_upper_bound(C) ? upper_bound(C) : nothing),
```
you don't need the qualification here as these functions are exported

Reply:
I wasn't 100% sure about the scope here since the keyword argument has the same name.

Reply:
True. Good catch